README.md

```diff
@@ -1,2 +1,34 @@
 Ethereum On-Boarding Testsuite
-==============================
\ No newline at end of file
+==============================
+
+## Implement a Single Node Ethereum Test Network
+
+1. Create an account on https://kurtosistech.com if you don't have one yet.
+2. Verify that the Docker daemon is running on your local machine.
+3. Clone this repository by running `git clone https://github.com/kurtosis-tech/kurtosis-onboarding-experience.git`
+4. Run the empty Ethereum single node test `my_test` to verify that everything works on your local machine.
+   1. Run `bash scripts/build-and-run.sh all`
+   2. Verify that the output of the build-and-run.sh script indicates that one test ran (my_test) and that it passed.
```

Comment:
Fun fact about Markdown: you actually don't need to make the numbers sequential for them to render correctly. For my Markdown documents, I just do:
```
1. First step
1. Second step
1. Third step
```
and it gets rendered as 1, 2, and 3 by the renderer